df.entities.xml

```diff
@@ -482,8 +482,7 @@
                 <enum base-type='int16_t' type-name='ethic_response'/>
             </static-array>
 
-            <static-array name='values' type-name='int32_t' count='32' index-enum='value_type'/>
-            <static-array name='values_2' type-name='int32_t' count='32' index-enum='value_type' since='v0.42.01'/>
+            <static-array name='values' type-name='int32_t' count='64' index-enum='value_type'/>
```

Comment:
These are still 64 elements, right?
Did you happen to notice anything in `values[33-63]`?